### ISSUE TITLE
About ESP Wireless Sensor Network

### ISSUE THREAD

**Issue by gpostolov** [honeycodeio/espy-mesh#4]
Hello,
Recently I'm trying to make a wireless sensor network through ESP8266. I wonder if you succeed in doing that?
Is it possible to carry out this type of network with ESP8266?
I would appreciate if you could help me with that!

Broadly speaking, the idea is to make a wireless sensor network(WSN) through the ESP8266.
For this idea I was looking the existence of any library for communication through a WIFI 802.11b protocol because this is support for the ESP8266, and create a WSN or a wireless mesh network (WMN).

In short, the idea is to create a network similar to the one presented here:

http://www.edn.com/electronics-products/electronic-product-reviews/other/4399007/WirelessHART---Internet-Protocol-wireless-sensor-networks-achieve-less-than-50-A-per-node

but using as nodes ("motes") the ESP8266.

Sincerely!
Gabriel Postolov
